README.md

```diff
@@ -1,55 +1,66 @@
-# License
+# common
```

Comment:
Should this be `commoners`? I'm not sure why there's a header called `common`

Reply:
+1